src/Disks/ObjectStorages/MetadataStorageFromPlainObjectStorage.h

```diff
@@ -117,11 +116,6 @@ class MetadataStorageFromPlainObjectStorageTransaction : public IMetadataTransac
 
     const IMetadataStorage & getStorageForNonTransactionalReads() const override;
 
-    void addBlobToMetadata(const std::string & /* path */, ObjectStorageKey /* object_key */, uint64_t /* size_in_bytes */) override
-    {
-        /// Noop
```

Comment:
This ignores the `WriteMode::Append` writes, which sounds very bad, since it is used in a few places in MergeTree code (at least), for example for `txn_version.txt`